_data/home.yml

```
@@ -0,0 +1,2 @@
+heading: Bring life back to your Pebble
+about: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse vehicula, purus vitae vulputate varius, ipsum orci maximus erat, nec tempor massa purus id purus. Nullam ligula augue, ultricies id sollicitudin quis, dictum sit amet leo.
```

```suggestion
about: On December 7th, 2016, <a href="https://web.archive.org/web/20180705003113/https://blog.getpebble.com/2016/12/07/fitbit/">Pebble announced that they were ceasing operations</a>.  Two days later, on December 9th, 2016, <a href="https://rebble.io/2016/12/09/rebble-pebble-reborn.html">the Rebble web site went live</a>, announcing our intention to "maintain and advance Pebble functionality, in the absence of Pebble Technology Corp."  We have fulfilled much of this goal via <a href="http://rebble.io/2018/07/01/rebble-era.html">Rebble Web Services</a>, which restores most of the functionality that the Pebble community knows and loves.  Read about our journey in the <a href="/blog/">blog</a>, find detailed instructions in our <a href="/howto/">HOWTO</a>, or simply...
```

(I might revise this later, but wanted to get you something ASAP so we can ship your amazing work!! ❣️ )